pontoon/actionlog/migrations/0002_auto_20200103_0313.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-01-03 03:13
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('actionlog', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='actionlog',
+            name='performed_by',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='actions', to=settings.AUTH_USER_MODEL),
+        ),
+    ]
```

Comment:
What's this? I don't understand where it's coming from (I don't see changes in the models that would justify it).

Reply:
When I rebased my code after the changes from your PR adding Black I was prompted to do the migration.

Reply:
Oh, crap. Sorry about that. It's a bug, please remove these files. I'm not sure what is triggering that warning though. :/

Reply:
Will do.